e2e/benchmarks/modelConfig.js

```diff
@@ -236,7 +236,7 @@ const benchmarks = {
           } else if (model.predict != null) {
             resultTensor = model.predict(inferenceInput);
           } else {
-            throw new Error("Predict function was not found.");
+            throw new Error('Predict function was not found.');
```

Comment:
should the custom model predictFunc be simplified with your newly created methods?